allocations/ipv6.md

```diff
@@ -13,6 +13,7 @@ Please add your allocation in lexicographical order.
 | 2001:db8:dead:beef:cafe:f00d:0::/112    | 2001:db8:dead:beef:cafe:f00d:0:0 - 2001:db8:dead:beef:cafe:f00d:0:ffff       | [Famicoman](https://github.com/Famicoman)|
 | 2001:db8:dead:beef:cafe:babe:0::/112    | 2001:db8:dead:beef:cafe:babe:0:0 - 2001:db8:dead:beef:cafe:babe:0:ffff       | [mikenabhan](https://github.com/mikenabhan)|
 | 2001:db8:dead:beef:cafe:be11:0::/112    | 2001:db8:dead:beef:cafe:be11:0:0 - 2001:db8:dead:beef:cafe:be11:0:ffff       | [darkdrgn2k](https://github.com/darkdrgn2k)|
+| 2001:db8:dead:beef::/112                | 2001:db8:dead:beef:0:0:0:0 - 2001:db8:dead:beef:0:0:0:ffff                   | [brannondorsey](https://github.com/brannondorsey)|
```

Comment:
It looks like this space was free,  so I claimed the first `/112` block in the subnet, but lmk if y'all would prefer I grab a different block. Or if I'm misunderstanding something about the reserved space of our IPv6 space.

Reply:
Looks good to me!